src/component/LevaPasswordInput.tsx

```
@@ -1,17 +1,31 @@
+import { useState } from 'react';
+import { Eye, EyeOff } from 'lucide-react';
 import { LevaInputProps } from 'leva/dist/declarations/src/types';
 import { Components, createPlugin, useInputContext } from 'leva/plugin';
 
+import './LevaPasswordInput.css';
+
 const { Label, Row, String } = Components;
```

Biome (2.1.2)

[error] Do not shadow the global "String" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)